resources/views/cases/casesMain.blade.php

```diff
@@ -0,0 +1,24 @@
+@extends('layouts.layout')
+
+<link href="{{ mix('css/tailwind.css') }}" rel="stylesheet">
```

Comment:
this line needs to be removed, in other PR this line is added in the layout.blade